src/borrower/feed/navfeed.sol

```diff
@@ -298,14 +299,15 @@ abstract contract NAVFeed is Auth, Discounting {
         loanDetails[loan].authWriteOff = true;
 
         bytes32 nftID_ = nftID(loan);
-        uint maturityDate_ = maturityDate(nftID_);
+        uint maturityDate_ = uniqueDayTimestamp(maturityDate(nftID_));
```

Comment:
same here

Reply:
I think here we only reading the maturity date to pass it to the `_writeOff` function.
We can just read it in `_writeOff`